### ISSUE TITLE
Walkie Talkies do not carry connect properly to voice server

### ISSUE THREAD

**Issue by Sukasa** [mekanism/Mekanism#1453]
I set up a server running Mekanism, and enabled voice chat on port 36123 mid-game after several clients had connected.  I forwarded the port in my router, and myself and two other players tried to communicate over the walkie talkies in-game.  We confirmed we were using the same channel in the walkie-talkies, and all were on, but no voice data got through.

Running netstat on the server computer showed connections between the server and the clients on the voice ports, with Recv-Q at ~50,000+ for each of them, and very low Send-Qs.

Disconnecting and reconnecting to the server (all players) did not alleviate the issue.

I completely exited my client and restarted, and the other players were able to hear me but I could not hear them.  However when a second player completely restarted their client (the server is on my LAN) it did not fix anything.

We conscripted a third player to help test at this point, and they could not hear me either, despite the second player able to hear me just fine.

tl;dr Mekanism voice chat was able to open a connection to the server, but voice generally did not make it through.
